drivers/usb/device/usb_dc_native_posix.c

```diff
@@ -0,0 +1,533 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief USB native_posix device driver
+ */
+
+#include <string.h>
+#include <stdio.h>
+#include <misc/byteorder.h>
+#include <usb/usb_dc.h>
+#include <usb/usb_device.h>
+#include <net/net_ip.h>
+
+#include "usb_dc_native_posix_adapt.h"
+
+#define LOG_LEVEL CONFIG_USB_DRIVER_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(native_posix);
+
+/* convert from endpoint address to hardware endpoint index */
+#define USBIP_EP_ADDR2IDX(ep)  ((ep) & ~USB_EP_DIR_MASK)
+/* get direction from endpoint address */
+#define USBIP_EP_ADDR2DIR(ep)  ((ep) & USB_EP_DIR_MASK)
+/* convert from hardware endpoint index and direction to endpoint address */
+#define USBIP_EP_IDX2ADDR(idx, dir)    ((idx) | ((dir) & USB_EP_DIR_MASK))
+
+#define USBIP_IN_EP_NUM		4
+#define USBIP_OUT_EP_NUM	4
+
+#define USBIP_MAX_PACKET_SIZE	64
+
+#define THREAD_STACK_SIZE	1024
```

Comment:
This is overkill, you can just set it to ARCH_POSIX_RECOMMENDED_STACK_SIZE

Reply:
done